recipes/zstd/all/conanfile.py

```diff
@@ -35,7 +30,7 @@ def config_options(self):
 
     def configure(self):
         del self.settings.compiler.libcxx
-        del self.settings.compiler.libstd
```

Comment:
interesting, we might need to improve hook to detect such misspellings

Reply:
not bad idea!

Reply:
Maybe it should be Conan itself?

Reply:
@jgsogo Yeah, why not. If a setting is being delete and is not listed in settings.yml, then we can raise an exception.